src/Persistence/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -64,15 +68,18 @@ private static void AddMinimalTemplates(ControlTemplateStore store)
                 }
             }
         });
-        store.Add(new() { Name = "commandComponent", Id = "http://microsoft.com/appmagic/CommandComponent" });
+        store.Add(new() { Name = "commandComponent", Id = WellKnownTemplateIds.CommandComponent });
     }
 
     /// <summary>
     /// Adds some default templates which are useful for testing
     /// </summary>
     /// <param name="store"></param>
-    private static void AddDefaultTemplates(ControlTemplateStore store)
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "TEST ONLY function")]
+    public static void TESTING_ONLY_AddDefaultTemplates(this ControlTemplateStore store)
```

Comment:
please mark this as `internal`. The project is already configured to use `InternalsVisibleTo` with the tests project

Reply:
there is a bug due to InternalsVisisbleTo not actually working due to someone disabling the wrong feature in SDK projects for this repo in order to implement versioning. We'll have to address it in the future.
[Bug 27469059](https://dev.azure.com/msazure/OneAgile/_workitems/edit/27469059): Maintainability: Repo versioning must use SDK proj properties; and reenable GenerateAssemblyGeneration

Note, this function will need to be called by the Doc Server Persistence tests too. Adding a prefix like the one here does ensure usage is clear.